benchmarks/docker-runner/scripts/test.js

```diff
@@ -0,0 +1,185 @@
+const yargs = require(`yargs`)
+const util = require('util')
+const { exit } = require('process')
+const exec = require('child_process').exec
+const execPromise = util.promisify(exec)
+
+const args = yargs
+  .option(`memory`, {
+    default: '2g',
+    describe: `The memory limit for the docker container`,
+  })
+  .option(`num-nodes`, {
+    default: '100',
+    describe: `The number of nodes to generate in the build`,
+  })
+  .option(`node-size`, {
+    default: '1k',
+    describe: `The rough size of each node generated in the build`,
+  })
+  .option(`command`, {
+    default: 'build',
+    choices: ['build', 'develop'],
+    describe: `The gatsby command to run for measurement`,
+  })
+  .option(`site`, {
+    default: process.cwd(),
+    describe: `The path to the site to use for testing`,
+  })
+  .option(`status`, {
+    default: 'cold',
+    choices: ['cold'],
+    describe: `The status of the build`,
+  }).parse()
+
+const hostExec = (cmd, promise = true) => {
+  console.log(`\x1b[2m$ ${cmd}\x1b[0m`)
+  return promise ? execPromise(cmd) : exec(cmd)
+}
+
+const getDockerExec = (dockerId) => {
+  return (cmd, env = {}, promise = true) => {
+    const envVars = Object.keys(env).map(key => `-e ${key}=${env[key]}`)
+    return hostExec(`docker exec ${envVars.join(' ')} ${dockerId} ${cmd}`, promise)
+  }
+}
+
+async function build(dockerExec) {
```

Comment:
The major change in this file was splitting out `build` and adding a `develop` option.